app/helpers/commodities_helper.rb

```diff
@@ -15,16 +15,16 @@ def commodity_level(commodity)
 
   def commodity_tree(main_commodity, commodities)
     if commodities.any?
-      content_tag(:ul, class: 'commodities') do
-        content_tag(:li) do
-          tree_code(commodities.first.code) + 
-          content_tag(:p, commodities.first.to_s.html_safe) +
-          tree_node(main_commodity, commodities, commodities.first.number_indents)
+      tag.ul(class: 'commodities') do
+        tag.li do
+          tree_code(commodities.first.code) +
+            tag.p(commodities.first.to_s.html_safe) +
+            tree_node(main_commodity, commodities, commodities.first.number_indents)
         end
       end
     else
-      content_tag(:ul, class: 'commodities') do
-        content_tag(:li, commodity_heading(main_commodity))
+      tag.ul(class: 'commodities') do
+        commodity_heading(main_commodity)
```

Comment:
This is the fix: remove the external `li` tag: the `commodity_heading` function already returns a `li` wrapper, and this "external" `li` tag was displayed above the other since HTML doesn't allow nesting a `li` inside another `li` directly.

The rest are Rubocop fixes.